src/styles/_contact.scss

```diff
@@ -8,6 +8,12 @@
     padding: 5px 10px;
   }
 
+  @media (min-width: grav-breakpoint(small)) {
+    h2 + div {
+      max-width: 83.33333%;
```

Comment:
whats this ... it seems a bit sketchy although I'm not sure what's this is doing? Could flex-box be a better option?

Reply:
Just copied the width that bootstrap was setting